.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Is it necessary to add this eclipse project and classpath files? If not remove them and create a .gitignore to exclude these file types to be included in this pr.